core/templates/dev/head/services/GenerateContentIdServiceSpec.ts

```diff
@@ -60,6 +44,7 @@ describe('GenerateContentIdService', function() {
 
   it('should throw error for unknown content id', function() {
     expect(function() {
+      // @ts-ignore
```

Comment:
This is intentional since we are explicitly passing a single parameter whereas the functions expects two paraments. We want that function to throw an error.

Reply:
You can add this description to top of this line.

Also, do you know why we are passing one argument here, I think passing two-argument will also work, right?